### DISCUSSION TITLE
Zoom mit Box - unterschiedliches Verhalten bei FireFox und Chrome

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#32] [Q&A]
WebGIS 5.23.203:
Chrome: 1. Ausführen "Sonnengang Abfrage" -> 2. Ergebnis wird in separatem Reiter dargestellt -> 3. Wechsel auf Reiter "Navigation" -> 4. Werkzeug "Zoom In" -> 5. Rahmen aufziehen -> 6. es wird auf Rahmen gezoomt.
FireFox: bis zu Schritt 5. alles gleich -> 6. meistens (leider nicht immer) wird eine neuerliche Sonnengang Abfrage durchgeführt (tritt übrigens woanders auch auf)

**Comment by gis-eni** [comment]
Ich kenn jetzt die "Sonnengang Abfrage" nicht. Kann ich das irgendwo nachvollziehen?
